### DISCUSSION TITLE
Struggling to Install

### DISCUSSION THREAD

**Discussion by svennm** [Eleven-Trading/TradeNote#48] [General]
Hello, I have been trying to run this app for some time, but I cannot get it to run can someone help?

**Comment by 7aklhz** [comment]
Can you tell me where in the installation process you are having difficulties? What errors do you see ? The easiest is to use discord to discuss this. You'll maybe find answers to your questions there as well

**Reply by 7aklhz** [reply]
Are your still having issues installing @svennm ?

**Comment by svennm** [comment]
I did not see you initial message, I am still unable to install. The
discord server link is broken. If you can hop on a discord call to help me
install I would appreciate it.

**Reply by 7aklhz** [reply]
This is the link on the Readme : https://discord.gg/ZbHekKYb85. For me it's working. Can you confirm that's working or not working ?

**Comment by 7aklhz** [comment]
This is the link on the Readme : https://discord.gg/ZbHekKYb85. For me it's working. Can you confirm that's working or not working ?

**Comment by svennm** [comment]
It and not working but this one did work thank you.

**Comment by gitricko** [comment]
Try https://github.com/gitricko/tradenote-selfhost
